src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static aliveAnimals() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+
+    return Animal.alive;
```

Comment:
The method `aliveAnimals` uses a filter inside a static method which is good for performance, as it avoids nesting loops. However, the name `aliveAnimals` could be misleading. Consider renaming it to something like `removeDead` to clearly indicate its purpose.